hr_time/hr_time_management/doctype/worklog/worklog.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2024, AtlasAero GmbH and contributors
+# For license information, please see license.txt
+
+from datetime import datetime
+import frappe
+from frappe.model.document import Document
+from frappe.model.docstatus import DocStatus
+
+class Worklog(Document):
+	def before_save(self):
```

Comment:
It should also be possible for admins or HR managers to create worklogs for other employees. The Employee field should therefore be displayed.
Normally it should already be automatically read-only for normal users with the current rights.

Reply:
I hid the field because I thought an employee seeing the employee column in his worklog list with only his own name did not make sense. I will unhide it. I will look for methods to display the field as read-only only for the original user or else editable for admins.